internal/utils/testdata/validate_compose_tests/valid/docker-compose.yml

```diff
@@ -88,7 +88,7 @@ services:
     - --target-peers=${CC_PEER_COUNT}
     - --execution-endpoints=${EC_AUTH_URL}
```

Comment:
While reviewing the docs, I noticed, that there is no `--execution-endpoints` flag, there is a singular version of it. Not sure how it works under the hood, but maybe it's worth updating this flag as well?